polaris-react/src/components/Banner/Banner.scss

```diff
@@ -4,28 +4,38 @@
   border: var(--p-border-width-1) solid var(--p-color-border-strong);
   background-color: var(--p-color-bg-app);
 
+  #{$se23} & {
+    border: none;
+    background-color: var(--p-color-bg);
+  }
+
   @if $in-page {
-    // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
-    @include focus-ring('wide');
```

Comment:
I already tested these changes in https://github.com/Shopify/polaris/pull/8780/files